.gitpod.yml

```diff
@@ -0,0 +1,44 @@
+image:
+  file: docker/dev-gitpod.Dockerfile
+
+# --------------------------------------------------------
+# exposing ports for liveserve
+ports:
+  - port: 5500
+    onOpen: notify
+
+vscode:
+  extensions:
+    - ms-python.python
+    - ritwickdey.liveserver
+
+tasks:
+  - init: |
+      # this needs to be a proj var
+      export PRE_COMMIT_HOME=/workspace/.precommit
```

Comment:
This should be removed before merge, but put into a Gitpod project environment variable.  Kept this here for illustration of how this works.